app/models/my_etm/featured_scenario.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module MyEtm
+  class FeaturedScenario < ActiveResource::Base
+    self.site = "#{Settings.identity.issuer}/api/v1"
+
+    def self.cached_ids
+      Rails.cache.fetch('featured_scenario_ids', expires_in: 1.week) do
+        connection.get('/api/v1/featured_scenarios/scenario_ids')
+          .body
+          .then { |json| JSON.parse(json)['scenario_ids'] }
```

Comment:
```suggestion
       find(:all, from: :scenario_ids)
```

I believe this does the exact same

Reply:
Plus we can grab the titles as well for clarity like we discussed!